actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/TenantMigrationActor.java

```diff
@@ -168,6 +184,13 @@ private void migrateUser(Request request, boolean notify) {
         reqMap, targetObject, correlatedObject, request.getContext());
   }
 
+  private Map<String, Object> userUpdateDeclaredRequest(String userId) {
+    Map<String, Object> requestMap = new HashMap();
+    requestMap.put(JsonKey.USER_ID, userId);
+    requestMap.put(JsonKey.STATUS, JsonKey.VALIDATED);
```

Comment:
The status is always validated? It could be rejected or anything else too. Can you take status a param.

Reply:
@indrajra : Further processing of other status are pending, i will take this in the next PR changes of SB-20024